sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/tests/Samples/Sample5_UpdateKnowledgeSources.cs

```diff
@@ -56,12 +60,12 @@ public void KnowledgeSources()
             {
                 Console.WriteLine(source);
             }
-            #endregion
+#endregion
```

Comment:
Nit: we typically align the `#region`s with the source, but it ultimately doesn't matter (just a bit easier to read, IMO).